src/rating/emoticon-rating.js

```diff
@@ -59,15 +77,25 @@ class EmoticonRating extends React.Component<
             previewIndex !== undefined ? x === previewIndex : x === value
           }
           $isSelected={x === previewIndex}
+          $isFocusVisible={this.state.isFocusVisible && isFocusable}
           onClick={() => this.selectItem(x)}
           onKeyDown={e => {
-            if (e.keyCode === SPACE_KEY_CODE || e.keyCode === ENTER_KEY_CODE) {
-              this.selectItem(x);
+            if (e.keyCode === ARROW_UP || e.keyCode === ARROW_LEFT) {
+              e.preventDefault();
+              const prevIndex = value - 1 < 1 ? 5 : value - 1;
```

Comment:
Would you mind adding a comment that `5` comes from the total number of items and is not customizable in the emoticon rating?

Reply:
```suggestion
              // 5 value comes from non-configurable number of icons
              const prevIndex = value - 1 < 1 ? 5 : value - 1;
```